FF1Lib/Classes/ClassesData.cs

```diff
@@ -349,27 +349,39 @@ public void BlackKnight(Flags flags)
 			{
 
 				var whiteslotsK = _spellPermissions[Classes.Knight].Where(s => ((int)s % 8) < 4).ToList();
-			
 				var whitetoblackslotsK = whiteslotsK.Select(s => (s + 4)).ToList();
-
 				var knightslots = whitetoblackslotsK.ToList();
 				_spellPermissions.ClearPermissions(Classes.Knight);
 				_spellPermissions.AddPermissionsRange(knightslots.Select(x => (Classes.Knight, x)).ToList());
-
+				var KnightLvl4Charge = new List<byte> { 0b00000000, 0b00000000, 0b00000000, 0b00000000, 0b00000000, 0b00000000, 0b00000000, 0b00000000, 0b00000000,
```

Comment:
so the same array is defined 3 times here, it should defined only once outside the function (but inside the classesdata class), something like
`private static List<byte> KnightLvl4Charge = new() { 0b00000000, etc. };`